inst/Upgrades.qml

```diff
@@ -37,14 +36,404 @@ Upgrades
 
 	Upgrade
 	{
-		functionName:		"ConfirmatoryFactorAnalysis"
+		functionName: 		"PrincipalComponentAnalysis"
+		newFunctionName:	"principalComponentAnalysis"
+		fromVersion:		"0.16.3"
+		toVersion:			"0.16.4"
+
+
+		ChangeRename
+		{
+			from:	"factorMethod"
+			to:		"componentCountMethod"
+		}
+		ChangeRename
+		{
+			from:	"parallelMethod"
+			to:		"parallelAnalysisMethod"
+		}
+		ChangeJS
+		{
+			name:		"parallelAnalysisMethod"
+			jsFunction:	function(options)
+			{
+				switch(options["parallelAnalysisMethod"])
+				{
+					case "pc":		return "principalComponentBased";
+					case "fa":		return "factorBased";
+				}
+			}
+		}
+		ChangeRename
+		{
+			from:	"eigenValuesBox"
+			to:		"eigenValuesAbove"
+		}
+		ChangeRename
+		{
+			from:	"numberOfFactors"
+			to:		"manualNumberOfComponents"
+		}
+		ChangeRename
+		{
+			from:	"basedOn"
+			to:		"analysisBasedOn"
+		}
+		ChangeJS
+		{
+			name:		"analysisBasedOn"
+			jsFunction:	function(options)
+			{
+				switch(options["analysisBasedOn"])
+				{
+					case "correlation":		return "correlationMatrix";
+					case "covariance":		return "covarianceMatrix";
+				}
+			}
+		}
```

Comment:
For some reason, this upgrade does not work. The radio button option is correctly renamed, but the values are not upgraded. Any idea @JorisGoosen ?